recipes/cmdstan/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "cmdstan" %}
+{% set version = "2.27.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stan-dev/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ff71f4d255cf26c2d8366a8173402656a659399468871305056aa3d56329c1d5
+
+build:
+  number: 0
+  binary_relocation: False
+
+requirements:
+  build:  
+    - {{ compiler('cxx') }}        # [not win]
+    - make                         # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - m2w64-make                   # [win]
+    - m2-filesystem                # [win]
+    - m2-sed                       # [win]
+    - m2-coreutils                 # [win]
+  host:
+    - tbb-devel =2020.2           # [not win]
```

To use the global pining
```suggestion
    - tbb-devel             # [not win]
```

---

Will the global pinning ever change? CmdStan can be compiled with more recent versions of TBB but it requires extra flags set in the make command

---

You'll get a PR to `cmdstan-feedstock` when it changes.

---

Great, I've removed the pinning. Thanks for all the review! Happy to address anything else that stands out still